desktop/src/preload/api/transfer/parseXlsxFileList.ts

```diff
@@ -20,7 +20,21 @@ export const parseXlsxFileList = (
               const accessionCell = sheet.B3?.v; // B3 cell value
               const applicationCell = sheet.B4?.v; // B4 cell value
 
-              if (accessionCell && applicationCell) {
+              if (
+                accessionCell &&
+                applicationCell &&
+                accessionCell.toString() !== "" &&
+                applicationCell.toString() !== "" &&
+                !Number.isNaN(
+                  Number(
+                    accessionCell
+                      .toString()
+                      .replace("-", "")
+                      .replaceAll(" ", "a")
+                  )
+                ) &&
+                !Number.isNaN(Number(applicationCell.toString()))
```

Comment:
Should we consider to create a helper function to perform the sanity check if both application num and accession num? The goal is to check both accessionCell and applicationCell are not empty and they are numeric, right?

Reply:
Great idea, added